src/main/java/com/serenitydojo/Dog.java

```diff
@@ -0,0 +1,27 @@
+package com.serenitydojo;
+
+public class Dog {
+    public String name;
+    public String favoriteToy;
+    public int age;
```

Comment:
I generally make fields private rather than public, to avoid them being modified accidentally. (It's a minor thing in test code but more important in application code.)

Reply:
Noted Sir! Thank You! 😃